app/connectors/MinimalDetailsConnector.scala

```diff
@@ -46,6 +46,7 @@ class MinimalDetailsConnectorImpl @Inject()(appConfig: FrontendAppConfig, http:
   )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[MinimalDetailsError, MinimalDetails]] =
     fetch("pspId", pspId.value)
 
+  // API 1442 (Get psa/psp minimal details) - https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=PODSP&title=PODS+API+Latest+Version
```

Comment:
Is it ok to have confluence links in the code?

Reply:
good shout! I think this is a public repo so i'll remove the links